compiler/ballerina-lang/src/test/java/io/ballerina/projects/test/resolution/packages/WorkspaceRepoDependencyTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
 *  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org).
 *
 *  WSO2 LLC. licenses this file to you under the Apache License,
```

Reply:
Fixed in [b472f53](https://github.com/ballerina-platform/ballerina-lang/pull/44229/commits/b472f5361e8761c1dc294d0ae9b472d4d4b884fe)